sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -787,6 +790,17 @@ static void _handle_rtr_adv(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
     if (dr == NULL) {
         return;
     }
+#if IS_ACTIVE(MODULE_DHCPV6_CLIENT)
+    if (rtr_adv->flags & NDP_RTR_ADV_FLAGS_M) {
+        if (IS_USED(MODULE_DHCPV6_CLIENT_IA_NA) &&
+            netif->ipv6.aac_mode & GNRC_NETIF_AAC_DHCP) {
```

Comment:
Shouldn't a set M flag imply, that `aac_mode` is set to `AAC_DHCP`?

Reply:
Fixed in e2f6fc2dc07ecaf8998f2144245875d1c3e3653f